electrum/lnchannel.py

```diff
@@ -1051,7 +1051,9 @@ def extract_preimage_from_htlc_txin(self, txin: TxInput) -> None:
             if is_sent:
                 self.lnworker.htlc_fulfilled(self, payment_hash, htlc.htlc_id, htlc.amount_msat)
             else:
-                self.lnworker.htlc_received(self, payment_hash)
+                # FIXME
+                #self.lnworker.htlc_received(self, payment_hash)
+                pass
```

Comment:
why is that commented out?

Reply:
> (the FIXME in lnchannel.py is pre-existing brokenness)

I just wanted to highlight that it is broken

Reply:
can you clarify?

Reply:
yes. `htlc_received` takes three required arguments but that call only provides one

Reply:
ah righht